app/models/meal.rb

```diff
@@ -15,11 +16,14 @@ class Meal < ApplicationRecord
   validates :image, size: { less_than: 5.megabytes }
   validates :price, numericality: { greater_than_or_equal_to: 0 }
 
+  ### CALLBACKS ###
+
   def set_non_veg
     update_column(:non_veg, ingredients.exists?(non_veg: true))
   end
 
   def set_price
     update_column(:price, ingredients_meals.sum(&:price))
+    line_items.each(&:update_total)
```

Comment:
we would have so many lineitems for completed orders in future.
Call this method only on line items of incomplete orders.

Reply:
Done